roles/builder/tasks/main.yml

```diff
@@ -1,4 +1,27 @@
 ---
+- name: Remove all images from storage
+  when: builder_image_storage_cleared
+  block:
+    - name: Get all images for removal
+      ansible.builtin.command: /usr/bin/composer-cli compose list
+      register: output
+      changed_when: false
+
+    - name: Remove each image by UUID
+      ansible.builtin.command: "/usr/bin/composer-cli compose delete {{ (item | split)[0] }}"
+      loop: "{{ output.stdout_lines }}"
+      changed_when: true
+
+- name: Get current image storage size
+  ansible.builtin.shell: "set -eo pipefail df -h /var/lib/osbuild-composer/artifacts | tail -n 1 | awk '{ print $5; }' | rev | cut -c2- | rev"
```

Comment:
this concerns me but I can't think of a better way to accomplish this ..... at a minimum we should add a [`failed_when`](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_error_handling.html#defining-failure) clause on it or else we could have this variable not contain an integer and the type cast below will fail in unpleasant ways.

Reply:
@maxamillion I agree lol I couldn't think of a better way. I was thinking about doing it in a python module but surprisingly it was easier in ansible. 

I'll add `failed_when` and write the docs